typings/custom-global/index.d.ts

```diff
@@ -1,23 +1,13 @@
+// @TODO https://github.com/microsoft/fluentui/issues/20544
+/// <reference path="../static-assets/index.d.ts" />
```

Comment:
as we fixed lage execution now `typings` project started to be run on CI which exposed global types errors ( duplication ) thus this needed to be mitigated. 

Pls see detailed issue to get better context https://github.com/microsoft/fluentui/issues/20544
cc @ecraig12345  @varholak-peter